javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,45 @@
+const width = 10;
+const height = 10;
+const area = width * height;
+
+console.log('value of area:', area);
+console.log('typeof area:', typeof area);
+
+const bill = 20;
+const payment = 30;
+const change = payment - bill;
+
+console.log('value of change:', change);
+console.log('typeof change:', typeof change);
+
+const quizzes = 90;
+const midterm = 96;
+const final = 80;
+const grade = (quizzes + midterm + final) / 3;
+
+console.log('value of grade:', grade);
+console.log('typeof grade:', typeof grade);
+
+const firstName = 'Andrew';
+const lastName = 'Roide';
+const fullName = `${firstName} ${lastName}`;
```

Comment:
Nice use of template literals. We won't be going over those yet... but if you use them correctly I won't complain :)